accel_x_max: 250.0
accel_x_min: -250.0
accel_y_max: 250.0
accel_y_min: -250.0
accel_z_max: 250.0
accel_z_min: -250.0
calibration_magn_use_extended: false
frame_id: imu_link
gyro_average_offset_x: -0.02
gyro_average_offset_y: -0.01
gyro_average_offset_z: 0.01
imu_yaw_calibration: 0.0
magn_ellipsoid_center:
- 0
- 0
- 0
magn_ellipsoid_transform:
- - 0
  - 0
  - 0
- - 0
  - 0
  - 0
- - 0
  - 0
  - 0
magn_x_max: -67.52
magn_x_min: -67.52
magn_y_max: -555.14
magn_y_min: -555.14
magn_z_max: -508.63
magn_z_min: -508.63
port: /dev/imu
